website/docs/core-concepts/organisations.md

```diff
@@ -4,4 +4,11 @@ sidebar_position: 8
 
 # Organisations
 
-## Coming Soon!
+To access Dega, creating an organization is mandatory. If Super Organisation is created while installing Dega, then the user can access Dega or else needs to create an organization from Kavach for having an access to Dega.
```

Comment:
Change to:

To access Dega, creating an organization is mandatory. The first organisation, created will be a `Super Organisation`. While a `Super Organisation` could be used as a normal organisation within a Dega instance, it has the following capabilities to manage the entire dega instance: 

- The users within the `Super Organisation` will have access to all the other organisations and their spaces. This is for support staff to help other organisations with any support queries.
- The admins within the `Super Organisation` will have admin access to the entire instance and will be able to manage the rest of the organisations and spaces. They have the ability to manage the usage limits for an oraganisation or spaces within.

If Super Organisation is created while installing Dega, the user can access Dega or else needs to create an organization from Kavach for having an access to Dega.